deno/run.sh

```diff
@@ -14,7 +14,14 @@ source /tmp/secrets_helper.sh
 echo "Done injecting Secrets, turning echoing back on"
 set -x
 
+# install the deno kernelspec, which we don't technically need since we're using a volume-mounted connection file
+deno --unstable \
+    jupyter \
+    --install
```

Comment:
This isn't adding silly startup slowdown?

Reply:
Yep you're right, moving it to the image build shaves off a bit over 0.5sec on kernel container start time 👍